stdlib/int64.gr

```diff
@@ -125,6 +235,15 @@ let abs = (n) => {
   WasmI64.sub(WasmI64.xor(n, mask), mask)
 }
 
+/**
+ * Computer the modulo of a division of its operands.
+ *
+ * @param x: The first operand
+ * @param y: The second operand
+ * @returns The modulo of a division of its operands
```

Comment:
Is it "modulo of a division" or is it just "a modulo"? I'm actually not sure.

Reply:
i dont know, i know it is bassically remainder of a division. just saying modulo would probably be fine though as well. im pretty sure it is just called modulo though.